test-suite/squidconf/bad-acl-http-status-dupe.conf

```diff
@@ -0,0 +1,18 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+acl test11 http_status 400 400
+acl test12 http_status 407 400 407
+acl test13 http_status 200 200-300
+
+acl test14 http_status 300-399
+acl test14 http_status 304
+
+acl test21 http_status 100-300 350 200-400
```

Comment:
What about 'reversed' ranges like 300-100? It looks like Squid treats this range as a single '300' value (regardless of PR changes), which is not intuitive, I think. It does not warn about them either. Similarly for ip ranges.